### PR TITLE
Week 1 Ruby-prep: finished exercises 05, 06, 14, and Tuesday exercise "Iteration".

### PR DESCRIPTION
**Pull request by AnvarArt**
Finished exercises 05, 06, 14, and Tuesday exercise "Iteration".

### CODE REVIEW

**Review by emperorliu — APPROVED** [review]
looks good!

File: lib/w1_Tuesday_Iteration.rb [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+	arr.each_with_index do |procedure, index|
+		runtime = benchmark(value)
+
+		if runtime < min_runtime
+			min_runtime = runtime
+			winner = index
+		end
+	end
+
+	arr[winner]
```
Comment (emperorliu) [inline lib/w1_Tuesday_Iteration.rb:22]: it looks like you might be missing an `end` here

Comment (emperorliu) [reply lib/w1_Tuesday_Iteration.rb:22]: actually this entire method seems to be indented improperly

Comment (AnvarArt) [reply lib/w1_Tuesday_Iteration.rb:22]: Yeah missed this one
Will put a fix to a next PR

**Pull request merged**